flatland/evaluators/trajectory_evaluator.py

```diff
@@ -5,7 +5,8 @@
 import tqdm
 
 from flatland.callbacks.callbacks import FlatlandCallbacks, make_multi_callbacks
-from flatland.trajectories.trajectories import Trajectory
+from flatland.envs.rail_env import RailEnv
+from flatland.trajectories.trajectories import Trajectory, SERIALISED_STATE_SUBDIR
 
 
 class TrajectoryEvaluator:
```

Comment:
refactor TrajectoryEvaluator by using PolicyRunner and compare*? We would only have one loop implementation left?